test/utils/include_args_to_hash_test.rb

```diff
@@ -0,0 +1,79 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Utils
+      class IncludeArgsToHashTest < Minitest::Test
+        def test_nil
+          input = nil
+          expected = {}
+          actual = ActiveModel::Serializer::Utils.include_args_to_hash(input)
+          assert_equal(expected, actual)
+        end
+
+        def test_empty_string
+          input = ''
+          expected = {}
+          actual = ActiveModel::Serializer::Utils.include_args_to_hash(input)
+          assert_equal(expected, actual)
+        end
+
+        def test_single_string
+          input = 'author'
+          expected = { author: {} }
+          actual = ActiveModel::Serializer::Utils.include_args_to_hash(input)
+          assert_equal(expected, actual)
+        end
+
+        def test_multiple_strings
+          input = 'author,comments'
```

Comment:
This would produce the expected output at this level, since this is just building a hash.